packages/create-llama/templates/python.ts

```diff
@@ -21,6 +22,13 @@ const getAdditionalDependencies = (vectorDb?: TemplateVectorDB) => {
       });
       break;
     }
+    case "pg": {
+      dependencies.push({
+        name: "llama-index",
+        version: "^0.9.19",
```

Comment:
the problem here is that we need to update now llama-index version in two places, here in in the original toml

Reply:
so i would leave the version out - and the updater below is using the the existing version

Reply:
how about this update @marcusschiesser https://github.com/leehuwuj/LlamaIndexTS/commit/5df2967065c6ab9e977a9a3634752d524983088b ? 
i implemented in an easy approach so it could be broken if we did not set the value in toml by a string of package version but in more complex toml scheme. In the long term, we may need a generic approach to merge dependencies in toml with the adding one in code.

Reply:
i think it should behave like this:
1. dependency doesn't exist: add it, but version must exist (if not throw an error)
2. if dependency does exist, update version and/or extras field

Reply:
as your `extras` approach is better, please also add it for MongoDB (if possible - and test it)

Reply:
@marcusschiesser i updated the code logic. I also checked the `llama-index` the pymongo is not a extras dependency on that so i think we should consider it as additional one in our code base.